README.md

```diff
@@ -191,7 +191,7 @@ In general, every development board has the following basic features:
         -  The parallel mode is a special feature, used for high-rated motors, requiring more than the 1.2A current limit.
         -  In this mode, the channel's directional pins and output pins are shorted; only one motor is connected to a motor driver i.e. two channels, giving a current capacity of 2.4A. Thus, one high rated motors can be controlled using ESP32.
         -  Note: The directional pin shorting is done by a manual DPDT switch. If the user turns on TB_A switch then the first motor driver goes into the parallel mode and its directional pins are shorted, where GPIO connections are IN1 = IN3 = 25 and IN2 = IN4 = 26. This is all done automatically. Also for parallel mode, the J1 and J2 junctions need to be shorted. 
-        <br/><br/>
+        -  Parallel mode has been discontinued in the current SRA Board.        <br/><br/>
```

Comment:
lgtm

Reply:
Please remove the parallel mode section from Readme.